src/reference/config/bindings/asyncapi/.partials/server.yaml

```diff
@@ -0,0 +1,7 @@
+  asyncapi_server:
+    type: asyncapi
+    kind: server
+    options:
+      specs:
+        my-mqtt-api-spec: mqtt/asyncapi.yaml
```

Comment:
This local file reference is not supported, right?
It should come from a catalog, perhaps `inline` or `filesystem`.